tests/test_settings.py

```diff
@@ -1,6 +1,7 @@
 """
 Validate that our settings functions work, and we can create yaml files
 """
+
```

Comment:
I had it in there, but it broke the tests.

`AssertionError: ((u'Admins', ''),) is not false` in `TestSettings.test_admin_settings`. 

It passes in Python 3, but fails in Python 2. I decided to stop messing with it for the sake of time.